peer.go

```diff
@@ -1468,6 +1468,17 @@ out:
 	for {
 		select {
 		case outMsg := <-p.sendQueue:
+			// If we're about to send a ping message, then log the
+			// exact time in which we send the message so we can
+			// use the delay as a rough estimate of latency to the
+			// remote peer.
+			if _, ok := outMsg.msg.(*lnwire.Ping); ok {
+				// TODO(roasbeef): do this before the write?
```

Comment:
TODO can be removed?

Reply:
i think this is slightly diff and related to getting more accurate measurements, but maybe @Roasbeef can comment if it's satisfied?